src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure {
+    private String color = ColorSupplier.assignColor();
+    private String property;
```

Comment:
What is property? Let's remove this field

Reply:
done